dbt/adapters/sqlserver/connections.py

```diff
@@ -40,11 +63,17 @@ def type(self):
     def _connection_keys(self):
         # return an iterator of keys to pretty-print in 'dbt debug'
         # raise NotImplementedError
-        return 'server', 'database', 'schema', 'port', 'UID', 'windows_login'
+        if self.windows_login is True:
+            self.authentication = "Windows Login"
+        
+    
+        return 'server', 'database', 'schema', 'port', 'UID', \
+                 'authentication', 'encrypt'
```

Comment:
So instead of displaying `windows_login` as True in `dbt debug` output , this changes would display `authentication: Windows Login`
We could have added windows auth as an Authentication value within the profiles config but
to have backward compatibility for Windows Auth with the config property : `windows_login ` otherwise `dbt debug` window would display `authentication=sql`